### PR TITLE
Enable reflow for all cartesian chart examples

### PR DESCRIPTION
**Pull request by krkshitij**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/979322ae432426868be11b010d88b9c1b37c0c0d'>979322ae432426868be11b010d88b9c1b37c0c0d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/AreaChart/AreaChart.types.ts [hunk 1]
```diff
@@ -59,6 +59,12 @@ export interface IAreaChartProps extends ICartesianChartProps {
    * Optimize area chart rendering for large data set.
    */
   optimizeLargeData?: boolean;
+
+  /**
+   * Prop to disable shrinking of the chart beyond a certain limit and enable scrolling when the chart overflows
+   * @default false
+   */
+  enableReflow?: boolean;
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/AreaChart/AreaChart.types.ts:67]: Isnt this happening automatically as it is part of IModifiedCartesian props.
We have recently added xAxisTitle, yAxisTitle, rotateXaxisLabels

Comment (krkshitij) [reply packages/react-charting/src/components/AreaChart/AreaChart.types.ts:67]: Extended prop types are not visible in the demo site, so I added this for all cartesian charts.

Comment (AtishayMsft) [reply packages/react-charting/src/components/AreaChart/AreaChart.types.ts:67]: Lets revert this for now. I will check why the extended props are not showing on demo site.

File: packages/react-examples/src/react-charting/LineChart/LineChart.CustomAccessibility.Example.tsx [hunk 2]
```diff
@@ -226,6 +226,7 @@ export class LineChartCustomAccessibilityExample extends React.Component<
           colorFillBars={colorFillBarData}
           allowMultipleShapesForPoints={this.state.allowMultipleShapes}
           enablePerfOptimization={true}
+          enableReflow={true}
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/LineChart/LineChart.CustomAccessibility.Example.tsx:229]: Doesn't linechart has enableReflow = true by default

Comment (krkshitij) [reply packages/react-examples/src/react-charting/LineChart/LineChart.CustomAccessibility.Example.tsx:229]: I added it for better clarity. Should I remove it from line chart examples?

Comment (AtishayMsft) [reply packages/react-examples/src/react-charting/LineChart/LineChart.CustomAccessibility.Example.tsx:229]: Yes we can remove.

Comment (krkshitij) [reply packages/react-examples/src/react-charting/LineChart/LineChart.CustomAccessibility.Example.tsx]: done

**Review by AtishayMsft — APPROVED** [review]
:shipit:

**Pull request merged**